connector/connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectStatFunctionSuite.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.connect.planner
+
+import scala.collection.JavaConverters._
+
+import org.apache.spark.connect.proto
+import org.apache.spark.sql.catalyst.dsl.expressions._
+import org.apache.spark.sql.connect.command.SparkConnectCommandPlanner
+import org.apache.spark.sql.connect.dsl.commands._
+import org.apache.spark.sql.test.{SharedSparkSession, SQLTestUtils}
+
+class SparkConnectStatFunctionSuite
```

Comment:
why do we need a new test suite?

Reply:
i encounter some problems in adding test in existing suites since summary requires a session and need to analyze the plan;
this suite will also cover some eargly computed stat functions (`cov` `corr`) in the future

Reply:
let me take another look

Reply:
removed, add another test in existing suites

Reply:
Initially I thought a separate suite was ok because that suite actually just executed the proto plan to check expected result. 

Now it's changed to plan comparison based test, which is ok.

Reply:
tests were improved a lot since the time i sent this pr :)